app/src/main/java/com/kickstarter/viewmodels/RewardViewHolderViewModel.kt

```diff
@@ -392,8 +392,8 @@ interface RewardViewHolderViewModel {
         /**
          * Use cases for enabling/disabling access to launch the next fragment
          * - If the selected reward has no addOns the CTA button will be enable if available
-         * - If the previously selected reward has addOns but not BackedAddOns CTA button available is reward available
-         * - If the previously selected reward has add-ons and has Backed addOns, CTA button available (they still can update the addOns selection)
+         * - If selecting other reward CTA button available if reward available
```

Comment:
does it still work if you're selecting the same sold out reward but just changing the add-ons?
The use case is, backers usually don't want to lose their early bird reward that's sold out but they want to add add-ons as creators add more of them throughout the campaign

Reply:
if selecting the same reward the button will be available for the user to be able to navigate to the AddOns Screen

Reply:
thanks for confirming!